src/Docusaurus/docs/tools/heat.md

```diff
@@ -34,15 +56,15 @@ The following properties control harvesting:
 | -------- | ----------- |
 | `HarvestDirectoryAutogenerateGuids` | Optional boolean property. Whether to generate authoring that relies on auto-generation of component GUIDs. The default is `$(HarvestAutogenerateGuids)` if specified; otherwise, **true**. |
 | `HarvestDirectoryComponentGroupName` | Optional string property. When harvesting multiple directories in a project, specify this metadata to create unique file names for the generated authoring. The component group name that will contain all generated authoring. |
-| `HarvestDirectoryDirectoryRefId` | Optional string property. The identifier of the Directory element that will contain all generated authoring. |
+| `HarvestDirectoryDirectoryRefId` | Optional string property. The identifier of the Directory into which the harvested files will be copied when you run the installer. |
```

Comment:
`be copied when you run the installer.` should be replaced with simply `installed.`

Reply:
I'm writing this with newbies in mind. I really struggled with the documentation and I'm suggesting changes that make it more understandable. Verbose and explicit was what I was aiming for.